Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
```

Comment:
### [coderay](http://coderay.rubychan.de)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.1 → 1.1.3

[_~~change-log~~, [source-code](http://github.com/rubychan/coderay), [gem-diff](https://my.diffend.io/gems/coderay/1.1.1/1.1.3)_]





<details>
<summary>Commits</summary>

A change of **110** commits. See the full changes on [the compare page](https://github.com/rubychan/coderay/compare/v1.1.1...v1.1.3).

These are the first 10 commits:
- (f06e0e3) [Add C++11 keywords](https://github.com/rubychan/coderay/commit/f06e0e319e13e0e4f1f90a7bdf634bc1ca66b182)
- (2612789) [Fixed typo](https://github.com/rubychan/coderay/commit/26127899acf888a1888195c8b50c4603167e2808)
- (92a8e56) [Small changes to grouping of new C++11 keywords](https://github.com/rubychan/coderay/commit/92a8e56ccd8d3aee05db9c7a821b24382528c674)
- (cee1223) [fix tests for Ruby 1.8](https://github.com/rubychan/coderay/commit/cee12238d0ca5ea0d9d1ae9ef0e4dbf80fdfc0dc)
- (2a4c37a) [Merge branch 'master' of https://github.com/rubychan/coderay](https://github.com/rubychan/coderay/commit/2a4c37a2a536e6d7b67ebe2ffe4d4f6a8fd1b42a)
- (e94cf8e) [fix Windows line breaks in FOLDERS file](https://github.com/rubychan/coderay/commit/e94cf8ea0e342f41837c4685a45c4d2af488fe6e)
- (dd142d2) [Fixed merge](https://github.com/rubychan/coderay/commit/dd142d29c860cb0c39ec30b25db6a8313310a09e)
- (54c94b7) [Add env setting to allow C extensions which are not allowed …](https://github.com/rubychan/coderay/commit/54c94b78f18939ef8517dd2818fbb699c0fa1391)
- (2bce2ac) [Allow failures with jruby 18 and 19 mode, RedCloth has a kno…](https://github.com/rubychan/coderay/commit/2bce2ac91ad7994f9ead130387c8b24905853064)
- (cd3bffb) [Shouldn't have removed the jruby lines](https://github.com/rubychan/coderay/commit/cd3bffb1086420c02774ba905a403f8efe313f46)


</details>